File: VB/dxSampleGrid/MyClasses/MyViewModel.vb
Imports System.Collections.ObjectModel

Namespace dxSampleGrid
    Public Class MyViewModel
        Public Sub New()
            CreateList()
        End Sub

        Public Property PersonList() As ObservableCollection(Of Person)

        Private Sub CreateList()
            PersonList = New ObservableCollection(Of Person)()
            For i As Integer = 0 To 4
                Dim p As New Person(i)
                p.ParentName = "LayoutGroup1"
                PersonList.Add(p)
            Next i
            PersonList(0).IsTabbed = True
            PersonList(1).IsTabbed = True
            PersonList(2).IsFloat = True
        End Sub
    End Class


End Namespace

File: CS/dxSampleGrid/MyClasses/MyViewModel.cs
using System.Collections.ObjectModel;

namespace dxSampleGrid {
    public class MyViewModel  {
        public MyViewModel() {
            CreateList();
        }
     
        public ObservableCollection<Person> PersonList { get; set; }

        void CreateList() {
            PersonList = new ObservableCollection<Person>();
            for (int i = 0; i < 5; i++) {
                Person p = new Person(i);
                p.ParentName = "LayoutGroup1";
                PersonList.Add(p);
            }
            PersonList[0].IsTabbed = true;
            PersonList[1].IsTabbed = true;
            PersonList[2].IsFloat = true;
        }
    }


}
